docs/src/docs/usage/false-positives.mdx

```diff
@@ -33,90 +34,83 @@ Exclude issue by text using command-line option `-e` or config option `issues.ex
 It's helpful when you decided to ignore all issues of this type.
 Also, you can use `issues.exclude-rules` config option for per-path or per-linter configuration.
```

Comment:
Perhaps `issues.exclude-rules` should be replaced with `linters.exclusions.rules`.